static/build/.tmp_update/script/update_networking.sh

```diff
@@ -14,6 +14,7 @@ update() {
     check_ntpstate
     check_httpstate
 	check_hotspotstate
+	sync_time &
```

Comment:
Should this be "backgrounded" ? What if you have a game that auto-starts - wouldn't you want to wait for NTP ?